app-admin/kubelet/kubelet-1.0.1.ebuild

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2015 CoreOS, Inc.. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+# $Header:$
+#
+
+EAPI=5
+
+DESCRIPTION="Kubernetes Container Manager"
+HOMEPAGE="http://kubernetes.io/"
+KEYWORDS="amd64 arm64"
+SRC_URI="https://storage.googleapis.com/kubernetes-release/release/v${PV}/bin/linux/${ARCH}/${PN}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+
+RDEPEND="net-misc/socat"
+
+src_unpack() {
+	mkdir ${S} || die
+	unpack ${A}
```

Comment:
also if $S is not the unpacked source you may as well set `S="${WORKDIR}"` above.